### PR TITLE
[DEV-8678] Status of Funds bar chart hover state

### PR DESCRIPTION
**Pull request by afnazu**
**High level description:**
Adds hovers to the Status of Funds bar chart that show users the actual Obligation and Total Budgetary Resource (TBR) amounts for each row of the existing sub-component and Federal Account layers in the drilldown.

**JIRA Ticket:**
[DEV-8678](https://federal-spending-transparency.atlassian.net/browse/DEV-8678)

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)

Reviewer(s):
- [x] Design review complete `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/_scss/pages/agencyV2/visualizations/_statusOfFundsChart.scss [hunk 1]
```diff
@@ -21,3 +21,54 @@
...
+    }
+    .tooltip__text-note {
+      font-size: rem(14);
+      font-style: italic;
+      text-align: center;
+      color: $color-base;
+    }
+    .tooltip__text-amount {
+      margin-left: auto;
+      font-weight: 600;
```
Comment (nick-torres) [inline src/_scss/pages/agencyV2/visualizations/_statusOfFundsChart.scss:54]: $font-semibold

File: src/_scss/pages/agencyV2/statusOfFunds/_visualizationSection.scss [hunk 2]
```diff
@@ -54,6 +54,7 @@
     h6 {
         font-size: rem(20);
         font-weight: 400;
```
Comment (nick-torres) [inline src/_scss/pages/agencyV2/statusOfFunds/_visualizationSection.scss:56]: $font-normal

File: src/_scss/pages/agencyV2/visualizations/_statusOfFundsChart.scss [hunk 3]
```diff
@@ -21,3 +21,54 @@
...
+}
+.sof_chart-tt {
+  @media (max-width: $medium-screen) {
+    display: none;
+  }
+}
+.sof-chart-tooltip {
+  .tooltip__title {
+    font-size: rem(16);
+    font-weight: 600;
```
Comment (nick-torres) [inline src/_scss/pages/agencyV2/visualizations/_statusOfFundsChart.scss:35]: $font-semibold

File: src/_scss/pages/agencyV2/statusOfFunds/_visualizationSection.scss (hunk 2 above)
Comment (afnazu) [reply src/_scss/pages/agencyV2/statusOfFunds/_visualizationSection.scss]: good catches thanks! fixed

**Pull request merged**